Docker/build/WDQS/runUpdate.sh

```diff
@@ -4,6 +4,6 @@
 cd /wdqs || exit
 
 # TODO env vars for entity namespaces, scheme and other settings
-/wait-for-it.sh "$WIKIBASE_HOST:80" -t 300 -- \
+/wait-for-it.sh "$WIKIBASE_HOST:$WIKIBASE_SCHEME" -t 300 -- \
```

Comment:
This is not making alot of sense to me, do you mean we should add WIKIBASE_PORT and use that here?

Reply:
Yeah, I missed the existence of variable WIKIBASE_PORT, it's a better fix.